### PR TITLE
[WIP][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[project_task_id_display]

### PR DESCRIPTION
**Pull request by dongshukai**
This PR is for module project_task_id_display
The module project_task_id and task_id_display has the same code and same function, they all repeate contains the task_id display and issue_id display. So I remove those 2 modules

### CODE REVIEW

File: project_task_id_display/README.rst [hunk 1]
```diff
@@ -2,41 +2,42 @@
...
 
- * have basic modules installed (project_issue)
-​
+ * have basic modules installed (project)
+
+
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline project_task_id_display/README.rst]: update according to https://github.com/Elico-Corp/odoo-addons/pull/171/files

File: project_task_id_display/__openerp__.py [hunk 2]
```diff
@@ -1,20 +1,17 @@
 # -*- coding: utf-8 -*-
-# © 2015 Elico corp (www.elico-corp.com)
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
-    'name': 'Task ID Display',
+    'name': 'Project Task ID Display',
     'version': '8.0.1.0.1',
```
Comment (elicoidal) [inline project_task_id_display/__openerp__.py]: 8.0.1.0.0

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
pls check

**Review by elicoidal — APPROVED** [review]

**Pull request merged**